lite/jupyter-lite.json

```
@@ -45,7 +45,7 @@
         "startMode": "single"
       },
       "@jupyterlab/notebook-extension:tracker": {
-        "autoStartDefaultKernel": true,
+        "autoStartDefaultKernel": false,
```

I guess you flipped it because you wanted to change kernel on runtime after opening the notebook; there is a better way to do it (it is always good to ask: is it a common thing? should I expect an API for it to already exist?) - see my comment suggesting to use `notebook:create-new` over `docmanager:new-untitled`.

Sorry that arguments for commands are not well documented upstream (yet! there is a WIP PR to change it).

```suggestion
        "autoStartDefaultKernel": true,
```